app/services/dunning_campaigns/create_service.rb

```diff
@@ -21,10 +21,7 @@ def call
             .update_all(applied_to_organization: false) # rubocop:disable Rails/SkipsModelValidations
 
           # NOTE: Stop and reset existing campaigns.
-          organization.customers.where(
-            applied_dunning_campaign_id: nil,
-            exclude_from_dunning_campaign: false
-          ).update_all( # rubocop:disable Rails/SkipsModelValidations
+          organization.customers.falling_back_to_default_dunning_campaign.update_all( # rubocop:disable Rails/SkipsModelValidations
```

Comment:
please, could you add the `with_dunning_campaign_not_completed` scope here? as you are already changing the code...